recipes-bsp/uefi/edk2-firmware-tegra/0001-Fix-eeprom-customer-part-numbers.patch

```diff
@@ -0,0 +1,83 @@
+diff -ru edk2-tegra_35.1.orig/edk2-nvidia/Silicon/NVIDIA/Drivers/EepromDxe/Eeprom.c edk2-tegra_35.1/edk2-nvidia/Silicon/NVIDIA/Drivers/EepromDxe/Eeprom.c
+--- edk2-tegra_35.1.orig/edk2-nvidia/Silicon/NVIDIA/Drivers/EepromDxe/Eeprom.c	2022-08-24 13:08:10.015078516 -0700
++++ edk2-tegra_35.1/edk2-nvidia/Silicon/NVIDIA/Drivers/EepromDxe/Eeprom.c	2022-09-16 16:46:38.880756491 -0700
+@@ -49,6 +49,9 @@
+   if (ChipID == T194_CHIP_ID) {
+     T194EepromData = (T194_EEPROM_DATA *)EepromData;
+     EepromBoardInfo = (TEGRA_EEPROM_BOARD_INFO *) BoardInfo;
++    if (T194EepromData->PartNumber.Leading[0] == 0xcc)

```

Comment:
We are using `0xcc` in different places in the code. Maybe we should create a `define` with a significant name